### DISCUSSION TITLE
jQuery version on 10.8

### DISCUSSION THREAD

**Discussion by gurvindergill** [owncloud/web#5888] [Q&A] [answered]
HI All,

I am testing 10.8 community version, would like to know what version jQuery is used? I cant find any documentation related to it.

**Comment by kulmann** [comment, marked as answer]
Hey, I just had a look in the network tab of my browser when visiting demo.owncloud.com (which is a 10.8 instance). That has jQuery in version v2.1.4 - hope this helps.
